File: backend/src/main/java/com/rsushe/weblab4/dto/PointResponse.java
package com.rsushe.weblab4.dto;

import lombok.*;

import java.time.Instant;
import java.time.LocalDateTime;

@NoArgsConstructor
@AllArgsConstructor
@Getter
@Setter
@Builder
public class PointResponse {
    private double x;
    private double y;
    private double radius;
    private boolean hit;
    private LocalDateTime creationDate;
    private Instant workingTime;
}

File: frontend/src/app/dto/PointResponse.ts
export class PointResponse {
    x: number;
    y: number;
    radius: number;
    hit: boolean;
    creationDate: Date;
    workingTime: number;

    constructor (_x: number, _y: number, _radius: number, _hit: boolean, _creationDate: Date, _workingTime: number) {
        this.x = _x;
        this.y = _y;
        this.radius = _radius;
        this.hit = _hit;
        this.creationDate = _creationDate;
        this.workingTime = _workingTime;
    }
}